### DISCUSSION TITLE
Do we expect another "hack-together"

### DISCUSSION THREAD

**Discussion by s0me1code** [microsoft/hack-together#83] [Q&A] [answered]
I just knew about the event. Do we expect another one or a similar one? 

And where could I find out about such events?

**Comment by aycabas** [comment, marked as answer]
Hi @A7medAbdien, we usually announce such events in [Microsoft 365 Developer blog](https://devblogs.microsoft.com/microsoft365dev/). It's not too late to join in this one, we still have 5 days in this hack, feel free to [register](https://aka.ms/hack-together/register), and even if you create one of the [Microsoft Graph scenarios](https://github.com/microsoft/hack-together/blob/main/top-scenarios.md) we provided, you can still earn a badge by submitting your project 🚀 Let us know if you need any guidance! 😊

**Reply by s0me1code** [reply]
Thank you🙏
